app/queries/validation_error_summary_query.rb

```diff
@@ -43,13 +61,17 @@ class ValidationErrorSummaryQuery
       COUNT(DISTINCT user_id) AS unique_users_all_time
     FROM validation_error_counts
     GROUP BY form_object, attribute
-    ORDER BY incidents_all_time DESC".freeze
+    #{order}".freeze
```

Comment:
Does `.freeze` make sense as this is dynamic now? (I'm not sure it made sense before).

Reply:
Yeah that can go - good spot